sqlmesh/core/macros.py

```diff
@@ -872,7 +872,7 @@ def star(
         for excluded in exclude.expressions or except_.expressions
     }
     quoted = quote_identifiers.this
-    table_identifier = alias.name or relation.name
+    table_identifier = normalize_identifiers(alias or relation, dialect=evaluator.dialect).name
```

Comment:
I think you need to get rid of the `.name` here and it should work?

Reply:
Yeah, it looks like my suggestion wasn't right after all, sorry about that.

> Not sure how to run the actual test suite though.

Did you already try `make fast-test`? I think you can also do `make doc-test` if the failing test is embedded in a docstring.

Reply:
I found the problem `alias or relation` was returning empty alias because even when alias not provided it is not None. So it needed `alias.name or relation.name` along with `normalize_identifiers().name`. 

Attempted to add a test too. Ran test_macros.py with success.